pyTigerGraph/gds/dataloaders.py

```diff
@@ -474,9 +466,7 @@ def _request_kafka(
             return
             
         while not exit_event.is_set():
-            status = tgraph._get(
-                tgraph.restppUrl + "/query_status", params=_stat_payload
-            )
+            status = tgraph.checkQueryStatus(resp["request_id"])
             if status[0]["status"] == "running":
                 sleep(1)
                 continue
```

Comment:
Might want to change the get function two lines below with getQueryResult as well?